src/main/java/io/anserini/search/SearchArgs.java

```diff
@@ -89,6 +89,18 @@
   @Option(name = "-f2log.s", metaVar = "[value]", required = false, usage = "F2Log s parameter")
   public float f2log_s = 0.5f;
 
+  @Option(name = "-sdm", usage = "boolean switch to use Sequential Dependence Model query")
+  public boolean sdm = false;
+
+  @Option(name = "-sdm.tw", metaVar = "[value]", usage = "term weight in sdm")
+  public float sdm_tw = 0.85f;
```

Comment:
"term weight in sdm" -> "SRM term weight"?
And below.

Reply:
SDM?